pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[build-system]
+requires = ["setuptools~=80.9", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "addonTemplate"
+dynamic = ["version"]
+description = "NVDA add-on template"
+maintainers = [
+	{name = "NV Access", email = "info@nvaccess.org"},
+]
+requires-python = ">=3.13,<3.14"
+classifiers = [
+	"Development Status :: 5 - Production/Stable",
+	"Intended Audience :: End Users/Desktop",
+	"License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+	"Operating System :: Microsoft :: Windows",
+	"Programming Language :: Python :: 3",
+	"Topic :: Accessibility",
+]
+readme = "readme.md"
+license = {file = "COPYING.TXT"}
+dependencies = [
+    "crowdin-api-client==1.21.0",
+    "lxml>=6.0.2",
+    "markdown>=3.10",
+    "markdown-link-attr-modifier==0.2.1",
+    "mdx-gh-links==0.4",
+    "mdx-truly-sane-lists==1.3",
+    "nh3==0.2.19",
+    "requests>=2.32.5",
+    "scons==4.10.1",
```

Comment:
are all of these required for the current state? or are some only necessary with #1 ?

Reply:
Except scons, they are required just for pr #1

Reply:
can you separate the ones only required for the submodule and #1, to those PRs